ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/StandaloneApiServer.scala

```diff
@@ -87,6 +88,8 @@ final class StandaloneApiServer(
   }
 
   private def buildAndStartApiServer()(implicit ec: ExecutionContext): Resource[ApiServer] = {
+    implicit val indexingOptions: IndexingOptions = IndexingOptions(implicitPartyAllocation = true)
```

Comment:
Please don't use an implicit for passing along a flag if there are less intrusive options available. The value of `IndexingOptions` is only read by `ActiveLedgerStateManager` and has a non-default value only for the sandbox. Can we pass the flag to its factory, please?